### ISSUE TITLE
Vbulletin Password capture

### ISSUE THREAD

**Issue by 001mofo** [kgretzky/evilginx2#47]
Firstly let me thank you for this work.  Its a fantastic project and very interesting.   My questions is this:
 
Vbulletin uses javascript to hash the password a few times before its submitted in a POST request.  any insight on how to intercept that javascript event and capture the plain text password as entered? 

`<form class="block vbform" method="post" action="login.php?do=login" onsubmit="md5hash(vb_login_password, vb_login_md5password, vb_login_md5password_utf, 0)">`

`function onsubmit(event) {
  md5hash(vb_login_password, vb_login_md5password, vb_login_md5password_utf, 0)
}`

**Comment by kgretzky** [comment]
I think for now it would only help to dynamically modify the vBulletin javascript and have it send `vb_login_password` value in an extra POST field, that Evilginx could intercept.

**Comment by kgretzky** [comment]
Closing for now - this could probably be solved when Javascript injection feature is implemented sometime in the future.

**Comment by 001mofo** [comment]
FYI 

I've got javascript injection working, but still was having issues getting the credentials....
I was able to capture credentials by oddly enough commenting out the md5 function for mimetype text/html.   But it still logs in to proxied site without any issues and captures the credentials, not cookies though.   I have a specific site in mind that I am working with that does not use 2 factor auth so not TOO concerned about the tokens.    

Is there a way to turn off the unauthorized redirect for short periods of time ?
I would gladly open a new issue if you'd prefer..

thanks again for your time :)
